src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -649,6 +649,40 @@ public void cannotCreateRequestWithoutValidUser()
       hasMessage("A valid user and patron group are required. User is null"))));
   }
 
+  @Test
+  public void cannotCreateRequestWithAnInactiveUser()
+    throws InterruptedException,
+    ExecutionException,
+    TimeoutException,
+    MalformedURLException {
+
+    final IndividualResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    final IndividualResource steve = usersFixture.steve();
+    final UUID pickupServicePointId = servicePointsFixture.cd1().getId();
+    final IndividualResource inactiveCharlotte = usersFixture.charlotte( c -> c.inactive());
+
+    loansFixture.checkOutByBarcode(smallAngryPlanet, steve);
+
+    DateTime requestDate = new DateTime(2017, 7, 22, 10, 22, 54, DateTimeZone.UTC);
+
+    final Response recallResponse = requestsClient.attemptCreate(new RequestBuilder()
+      .recall()
+      .forItem(smallAngryPlanet)
+      .withPickupServicePointId(pickupServicePointId)
+      .withRequestDate(requestDate)
+      .withRequesterId(inactiveCharlotte.getId()));
+
+    assertThat(recallResponse, hasStatus(HTTP_VALIDATION_ERROR));
+
+    assertThat(recallResponse.getJson(), hasErrorWith(allOf(
+      hasMessage("Inactive users cannot make requests"))));
+
```

Comment:
Are there intended to be blank lines between these checks?

Reply:
I don't have strong opinions about these blank lines.

Reply:
I'd prefer they were removed, so each check was on the next line after each other

Reply:
I combined them into one assertion and removed the blank lines.